### PR TITLE
add requested icons

### PR DESCRIPTION
**Pull request by timbuktuuu**
the missed PR

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -171,6 +171,24 @@ define(function (require, exports, module) {
 	addIcon('yml',   'ion-navicon', '#008000');
 	addAlias('yaml', 'yml');
 	addIcon('sqf',   'ion-wand', '#b9e11f');
+    addAlias('map', 'json');
```
Comment (ivogabe) [inline main.js]: Can you fix the indentation here?

**Comment by ivogabe** [comment]
Can you add files to the examples directory, for each new addIcon call (not addAlias)? Also, I'd prefer different colors for these icons, can you choose some colors for them? Logo's and websites can give some inspiration.

**Comment by timbuktuuu** [comment]
done but can't find something for .re was just for the Issues

**Comment by ivogabe** [comment]
I also couldn't found .re, I think it's best to remove it.

**Comment by timbuktuuu** [comment]
here you go!

**Comment by timbuktuuu** [comment]
oops

**Comment by ivogabe** [comment]
Can you add new extensions (only the addIcon calls) to the examples directory?

**Comment by timbuktuuu** [comment]
ready 2 merge?

**Comment by ivogabe** [comment]
Yes! Thanks

**Pull request merged**